criu/include/linux/mount.h

```diff
@@ -6,7 +6,7 @@
 
 /* Copied from /usr/include/sys/mount.h */
 
-#ifndef FSCONFIG_CMD_CREATE
+#ifndef FSOPEN_CLOEXEC
```

Comment:
why is FSOPEN_CLOEXEC better than FSCONFIG_CMD_CREATE?

Reply:
The comment above indicates that this part has been copied from `sys/mount.h`. In a recent [commit](https://sourceware.org/git/?p=glibc.git;a=commitdiff;h=774058d72942249f71d74e7f2b639f77184160a6) was added the `ifdef` condition for `FSOPEN_CLOEXEC` to `sys/mount.h`. Thus, here we replace `FSCONFIG_CMD_CREATE` with `FSOPEN_CLOEXEC` for consistency with `/usr/include/sys/mount.h`